sencha-workspace/packages/slate-assets/src/assets/controller/Assets.js

```diff
@@ -376,6 +376,7 @@ Ext.define('Slate.assets.controller.Assets', {
         Ext.Ajax.request({
```

Comment:
@KClough also all those `asset.` calls can be done at once: `asset.set('Stories', responseData.data, { commit: true })`